htdocs/web_portal/views/downtime/downtimes_overview.php

```diff
@@ -25,11 +26,17 @@
         </div>
         <div style="float: left;">
             <h1 style="float: left; margin-left: 0em; padding-bottom: 0.0em;">
-                   Overview of Current and Planned Downtimes
+                   Downtimes
             </h1>
             <span style="float: left; clear: both;">
-                All currently active and planned downtimes over the coming weeks<br />
-             </span>
+                Currently active and scheduled downtimes<!-- This comment removes the pad space
+                --><?php
+                    if ($filterScope != NULL) {
+                        echo ', filtered by scope "' . $filterScope . '"';
+                    }
+                ?><!-- This comment removes the pad space
+                -->.<br />
+            </span>
```

Comment:
I'm torn on this change, but I think i prefer it as before. Could you revert this chunk?

Reply:
Reverted. I must kick the habit of trying to make things better ;-)